tutorials/hana-cloud-mission-trial-9/hana-cloud-mission-trial-9.md

```diff
@@ -3,26 +3,27 @@ parser: v2
 author_name: Christopher Kollhed
 author_profile: https://github.com/chriskollhed
 auto_validation: true
-time: 10
+time: 15
 tags: [ tutorial>beginner, products>sap-hana-cloud, software-product-function>sap-hana-cloud\,-sap-hana-database, products>sap-business-application-studio]
 primary_tag: products>sap-hana-cloud
```

Comment:
Please add "parser: v2" as the last item in the meta data section, below the primary tag.